straxen/get_corrections.py

```diff
@@ -37,72 +38,50 @@ def correction_options_wrapped(run_id, conf, *arg):
 
 @export
 @correction_options
-def get_to_pe(run_id, gain_model, n_pmts):
-    if not isinstance(gain_model, tuple):
-        raise ValueError(f"gain_model must be a tuple")
-    if not len(gain_model) == 2:
-        raise ValueError(f"gain_model must have two elements: "
-                         f"the model type and its specific configuration")
-    model_type, model_conf = gain_model
-
-    if model_type == 'disabled':
-        # Somebody messed up
-        raise RuntimeError("Attempt to use a disabled gain model")
-    if model_type == 'CMT_model':
-        if not isinstance(model_conf, tuple) or len(model_conf) != 2:
-            # Raise a value error if the condition is not met. We should have:
-            # ("CMT_model", -> To specify that we want to use the online
-            #                  corrections management tool
-            #   (
-            #   "to_pe_model", -> This is to specify that we want  the gains
-            #   "v1", -> The version of the correction 'v1' is the online version
-            #   )
-            raise ValueError('CMT gain model should be similar to:'
-                             '("CMT_model", ("to_pe_model", "v1"). Instead got:'
-                             f'{model_conf}')
-        # is this the best way to do this?
-        is_nt = n_pmts == straxen.n_tpc_pmts or n_pmts == straxen.n_nveto_pmts or n_pmts == straxen.n_mveto_pmts
-
-        corrections = straxen.CorrectionsManagementServices(is_nt=is_nt)
-        to_pe = corrections.get_corrections_config(run_id, model_conf)
-
-        return to_pe
-
-    elif model_type == 'to_pe_per_run':
-        warn("to_pe_per_run will be replaced by CorrectionsManagementSevices",
-             DeprecationWarning, 2)
-        # Load a npy file specifying a run_id -> to_pe array
-        to_pe_file = model_conf
-        x = straxen.get_resource(to_pe_file, fmt='npy')
-        run_index = np.where(x['run_id'] == int(run_id))[0]
-        if not len(run_index):
-            # Gains not known: using placeholders
-            run_index = [-1]
-        to_pe = x[run_index[0]]['to_pe']
-
-    elif model_type == 'to_pe_constant':
-        if model_conf in FIXED_TO_PE:
-            return FIXED_TO_PE[model_conf]
-
-        try:
-            # Uniform gain, specified as a to_pe factor
-            to_pe = np.ones(n_pmts, dtype=np.float32) * model_conf
-        except np.core._exceptions.UFuncTypeError as e:
-            raise (str(e) +
-                   f"\nTried multiplying by {model_conf}. Insert a number instead.")
-    else:
-        raise NotImplementedError(f"Gain model type {model_type} not implemented")
+def get_to_pe(run_id, conf, n_pmts):
```

Comment:
I think we can join this function with `get_correction_from_cmt`. See my comments below.

Reply:
agree